packages/base/any/kernels/5.10-lts/patches/0025-kernel-5-10-4-marvell10G-support-2500-BaseT.patch

```diff
@@ -0,0 +1,58 @@
+From b7dc1d4800a9d2445cb53426839541363398d20a Mon Sep 17 00:00:00 2001
+From: paulchen040968 <paul.chen@wtmec.com>
+Date: Tue, 25 May 2021 23:12:06 +0800
+Subject: [PATCH 4/4] kernel 5.10.4 marvell10G support 2500_BaseT
```

Comment:
1.  Where are the first three patches from the series? This only raises questions.
2.  Please follow the guide [*How to Write a Git Commit Message*](https://chris.beams.io/posts/git-commit/).
3.  Please add a Signed-off-by line.